src/main/java/core/basesyntax/Engine.java

```diff
@@ -1,5 +1,67 @@
 package core.basesyntax;
 
-public class Engine {
+public class Engine implements Cloneable {
+    private String brand;
+    private String model;
+    private int volume;
+    private int power;
+    private int numberOfCylinders;
 
+    public Engine(String brand, String model, int volume, int power, int numberOfCylinders) {
+        this.brand = brand;
+        this.model = model;
+        this.volume = volume;
+        this.power = power;
+        this.numberOfCylinders = numberOfCylinders;
+    }
+
+    /**
+     * Хотя в задании сказано не использовать super.clone(), поскольку объект
+     * содержит только примитивы и стринги, вполне правильной будет такая реализация
+     * Богдан в слаке сказал, что это требование относится к классу Car.
+     */
```

Comment:
you should better leave comments in GitHub when creating your pull request
this is a Javadoc comment, and it is intended to provide a description for a complex method

Reply:
ok, will be doing this way!